main.go

```diff
@@ -5,6 +5,7 @@ import (
 	"os"
 	"time"
 
+	audittrail "github.com/spacelift-io/spacectl/internal/cmd/audit_trail"
```

Comment:
could we rename the dir to `audittrail`? I'm not a fan of named imports :/

Reply:
Sure, I will change that. I'm also not a fan of named imports. I did that only because this is how run-external-dependency command was done.